Python/import.c

```diff
@@ -99,6 +99,71 @@ _PyImportZip_Init(PyThreadState *tstate)
     return _PyStatus_ERR("initializing zipimport failed");
 }
 
+
+/* Debugging functions */
+
+// Check if message starts with keyword
+// Return True if keyword is the prefix of message
+static bool
+_startswith(const char *message, const char *keyword)
+{
+    size_t len_message = strlen(message);
+    size_t len_keyword = strlen(keyword);
+
+    if (len_message < len_keyword) {
+        return false;
+    }
+
+    // check if the prefix of message matches keyword or not
+    for (size_t i = 0; i < len_keyword; i++) {
+        if (keyword[i] != message[i]) {
+            return false;
+        }
+    }
+    return true;
+}
+
+// Print the message to stderr if -v/PYTHONVERBOSE is turned on
+static void
+_verbose_message(const char *message, const char *args)
+{
+    PyThreadState *tstate = _PyThreadState_GET();
+    int verbose = _PyInterpreterState_GetConfig(tstate->interp)->verbose;
+
+    if (verbose) {
+        // confirm the prefix of message matches any keyword
+        bool startswith_keywords = false;
+        char keywords[2][8] = {
+                                "#",
+                                "import "
+                            };
+
+        for (int idx = 0; idx < 2; idx++) {
+            char *keyword = keywords[idx];
+
+            if (_startswith(message, keyword)) {
```

Comment:
Instead of implementing `_startswith`, you could use `strncmp`, and it'd simplify things a bit. Something like:

```c
if (strncmp(message, keyword, strlen(keyword)) == 0) {
  ...
}
```

Reply:
And since the list is so short, I'd go as far as maybe doing it in a single `if`, something like:

```c
if (strlen(message) > 1 && (
      message[0] == '#' || strncmp(message, "import ", 7) == 0
)) {
    fprintf (stderr, message, args);
}
else {
    ...
}
```

...and reduce the complexity.

Reply:
I was thinking about using an array that will be easier to be extended in the future, but the keywords seem pretty fixed in this case. Thus, it would also be nice to use if/else conditions.